marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/matching.cy.tsx

```diff
@@ -551,6 +551,23 @@ describe("Matching", () => {
       // To test compare values for matched Uris
       cy.log("**To test compare values for matched Uris**");
       cy.findAllByLabelText("/json/persons/first-name-double-metaphone compareButton").first().scrollIntoView().click();
+      cy.waitForAsyncRequest();
+      // Check  column Order
+      cy.get(".modal-body")
+        .within(() => {
+          cy.get("th").eq(2).within(() => {
```

Comment:
I think the validated scenario here is flipped. It should be (in pseudo):
th (2) = Current Document: "/json/persons/first-name-double-metaphone1.json"
th (3) = Current Document: "/json/persons/first-name-double-metaphone2.json"
th (4) = Preview:

Reply:
Fixed ✅